src/platform/packages/shared/kbn-test/src/functional_test_runner/lib/mocha/reporter/scout_ftr_reporter.ts

```diff
@@ -101,7 +99,7 @@ export class ScoutFTRReporter {
     return {
       path: filePath,
       owner: fileOwners,
-      area: this.getOwnerAreas(fileOwners),
+      area: this.getOwnersGroups(fileOwners),
```

Comment:
```suggestion
      group: this.getOwnersGroups(fileOwners),
```

Reply:
@dolaru This was affecting property names that are reported to an external Kibana, that's why I did not change them straight away. I would like to align with your team on this subject. I can rollback the "Areas vs Groups" changes for now and we can tackle this as a follow-up if you prefer.

Reply:
> Scout is still in a "pre-release" stage at the moment, so breaking changes aren't an issue.

Good to know! In that case I can update it to 'group' so that everybody uses the same terminology.

> Either way, for the foreseeable future, we don't expect a need to stray from these new group definitions in the area/group field in Scout.

Well, to that regard, I met with @pheyos and @dmlemeshko a few weeks ago, and we were thinking of writing a script that consumes this data and helps us automate the test categorisation a little bit. But I haven't had the time to make any progress yet.